server/lib/pbench/test/unit/test_process_tb.py

```diff
@@ -1,67 +1,75 @@
+import os
 import logging
 import pytest
 
-from pbench import get_pbench_logger
+from pathlib import Path
+
 from pbench.process_tb import ProcessTb, Results
-from pbench.test import tarball, bad_tarball, receive_dir
+from pbench.test import tarball, bad_tarball, receive_dir, fixture_path
 
 
 class MockConfig:
-    def __init__(self):
-        self.TS = "run-1970-01-02T00:00:00.000000"
+    TS = "run-1970-01-02T00:00:00.000000"
 
     def get(self, section: str, option: str):
         pass
 
 
 class TestProcessTb:
     @pytest.fixture(autouse=True)
-    def config_and_logger(self, valid_config):
+    def logger(self, test_logger):
         # Setup the configuration and logger
-        global tmp
-        config, tmp = valid_config
-        self.logger = get_pbench_logger("pbench", config)
+        self.logger = test_logger
 
     def test_get_receive_dir_failed(self, monkeypatch):
         def mock_get(section: str, option: str):
             if option == "pbench-receive-dir-prefix":
                 return ""
+            assert False
```

Comment:
I believe that you can add explanatory text to the assertion, which would be a good idea since `assert False` is not particularly illuminating when it triggers:
```
            assert False, f"Unexpected configuration option, {section!r}/{option!r}, in mocked config.get()"
```

(Ditto for the other, bare `assert False` occurrences.)